src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,38 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Event class that extends Task, by adding a new parameter called "at", to dictate when the event is happening
+ */
+public class Event extends Task {
+
+    /**
+     * Variable to store when the Event is happening at
+     */
+    public LocalDateTime at;
+
+    /**
+     * Constructor for Event object, specifying the description and datetime at which the event is happening
+     * @param description Description of the Event
+     * @param at String representation of Date & Time at which the event is happening
+     */
+    public Event(String description, String at) {
+        super(description);
+        DateTimeFormatter inputDtf = DateTimeFormatter.ofPattern("d/M/yyyy HHmm");
```

Comment:
You might want to use DateTime instead of shortening it as Dt
Food for thought. How will you name a variable with the word UTC? utc, UTC, Utc?